sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 **Breaking Changes**
 * Passing any type other than `ReceiveMode` as parameter `receive_mode` now throws a `TypeError` instead of `AttributeError`.
+* `AMQPMessage` (`Message.amqp_message`) properties are now read-only (will not be reflected in the underlying message) pending cross-language sdk design consensus.
```

Comment:
Do we want to mention this in the changelog? -- `(will not be reflected in the underlying message) pending cross-language sdk design consensus.`

Reply:
Mmm.  I agree, but unfortunately, this WAS my attempt at clarification XD

(previously I just had "properties are now read-only" but if you hit this as a user you'll find that you can modify them, because python doesn't like, have a concept of private/readonly dict, it just won't "do anything."  Am very open to better verbiages.)

Reply:
```suggestion
* `AMQPMessage` (`Message.amqp_message`) properties are now read-only, changes of these properties would not be reflected in the underlying message.  This may be subject to change before GA.
```
(per our OOB chat about this)